Makefile

```diff
@@ -79,6 +79,7 @@ help:
 	@echo "    LEPTONICA_VERSION  Leptonica version. Default: $(LEPTONICA_VERSION)"
 	@echo "    TESSERACT_VERSION  Tesseract commit. Default: $(TESSERACT_VERSION)"
 	@echo "    TESSDATA_REPO      Tesseract model repo to use. Default: $(TESSDATA_REPO)"
+	@echo "    TESSDATA      	  Path to the .traineddata directory to start finetuning from. Default: $(LOCAL)/share/tessdata
```

Comment:
Please use space characters here instead of TAB.

Line 9 also still needs a comment now.